gui/encyclopedia/articles/ancient_world/warfare/crossbow_designs.json

```diff
@@ -0,0 +1,5 @@
+{
+    "title": "Different Crossbow Designs",
+    "subtitle": "Two contrasting philisophies and their characteristics",
+"content":"The strength and power of a crossbow primarily depends on the energy required to span it (i.e., the total energy it stores in full draw). And the latter is directly proportional to two physical quantities - the length of its powerstroke as well as its total draw weight; the stronger and longer a crossbow has to be drawn, the faster it can launch an arrow (obviously).\nHowever, this also translates to another correlation: the smaller one of these quantities, the higher the other has to be in order to prevent loss in overall shooting power; the shorter the powerstroke of a crossbow, the higher the total draw weight it needs to compensate for it (to achieve the same output).\nAsian (mainly Chinese) crossbows featured a comparably long draw length and powerstroke of over 20 inches (50 centimeters). This allowed keeping the draw weight low enough for it to be spanned by simply pulling back the bowstring with the hands (without extra spanning devices).\nMedieval European steel crossbows, on the other hand, oftentimes had only half as long a powerstroke and, in turn, gigantic draw weights of up to multiple thousand pounds. Obviously, no soldier was able to draw them by themselves, which is why they developed various spanning devices, ranging from simple drawing belts to complex winch systems.\nBoth European and Asian crossbow designs result in advantages and disadvantages. The Chinese crossbow's long powerstroke allowed long (therefore stable-in-flight) and precise arrows to be used. Additionally, the simple spanning motion maximized reload speeds. The only big downside of this design laid in the cumbersome length of the crossbow's prod (bow), which was necessary to achieve such long powerstrokes. However, accuracy and firing speed factored out this bulkiness on the wide, open battlefields of ancient China.\nThe opposite was the case in medieval Europe, where castle and siege warfare dominated. While European crossbows are often regarded as less energy-efficient, slower, and less accurate (due to the very short crossbow bolts), they proved handier and perfectly suited the need for shooting at close range off battlements and fighting in tight formations."
```

Comment:
```suggestion
"content":"The strength and power of a crossbow primarily depends on the energy required to span it (i.e., the total energy it stores in full draw). And the latter is directly proportional to two physical quantities - the length of its powerstroke as well as its total draw weight; the stronger and longer a crossbow has to be drawn, the faster it can launch an arrow (obviously).\nHowever, this also translates to another correlation: the smaller one of these quantities, the higher the other has to be in order to prevent loss in overall shooting power; the shorter the powerstroke of a crossbow, the higher the total draw weight it needs to compensate for it (to achieve the same output).\nAsian (mainly Chinese) crossbows featured a comparably long draw length and powerstroke of over 20 inches (50 centimeters). This allowed them to keep the draw weight low enough for soldiers to span it by pulling back the bowstring back with  hands (without extra spanning devices).\nMedieval European steel crossbows, on the other hand, oftent had only half as long a powerstroke and, in turn, gigantic draw weights. Obviously, no soldier was able to draw them unaided, which is why they developed various spanning devices, ranging from simple drawing belts to complex winch systems.\nBoth European and Asian crossbow held different advantages and disadvantages. The Chinese crossbow's long powerstroke allowed long  and precise arrows to be used. Additionally, the simple spanning motion maximized reload speeds. The only big downside of this design lay in the cumbersome length of the crossbow's prod (bow), which was necessary to achieve such long powerstrokes. However, accuracy and firing speed factored out this bulkiness on the wide, open battlefields of Ancient China.\nThe opposite was the case in medieval Europe, where castle and siege warfare dominated. While European crossbows are often regarded as less energy-efficient, slower, and less accurate (due to the very short crossbow bolts), they perfectly suited the need for shooting at close range off battlements and fighting in tight formations."
```